### DISCUSSION TITLE
Is it possible to search for multiple items of the same type in a single search? (ie. two different FROM addresses)

### DISCUSSION THREAD

**Discussion by gregdotca** [umanamente/n8n-nodes-imap#10] [Q&A]
I'm currently setting up a priority email notification system, where I check an IMAP inbox for certain people and if there are emails from them, it sends me a push notification, and I'm wondering if it's possible to search for multiple FROM addresses in a single search?

This works perfectly if I search for a single email address using the "From Contains" Search Filter, but I can't seem to find a way to have it search for multiple email addresses using something like | OR , etc., which makes it so that instead of just typing an additional email address to find emails from, I need to duplicate and update multiple nodes in my workflow, which gets out of hand fairly quickly if you want to search for more than just a few addresses.

**Comment by umanamente** [comment]
Unfortunately it is not possible currently.
I would suggest using a list node before IMAP node containing email addresses, that will execute IMAP node for each email in the list.

**Comment by horst789github** [comment]
+1 any news here?
